src/main/java/homeworks/jose/secondAssigment/ClassD.java

```diff
@@ -0,0 +1,10 @@
+package homeworks.jose.secondAssigment;
+
+public class ClassD extends Father{
+    private Map<,> d;
+
+    @Override
+    public void methodC() {
+        super.methodC();
```

Comment:
there is no need for an override (and therefore the method) since the behavior of both methods are the same